source/_includes/pup43_platforms_fedora.markdown

```diff
@@ -0,0 +1,7 @@
+We publish official packages and run automated testing on the following versions for Puppet 4.3:
+
+* Fedora 22
+* Fedora 21
+* Fedora 20
```

Comment:
What's up with http://yum.puppetlabs.com/puppetlabs-release-pc1-fedora-20.noarch.rpm, then? Is it no longer being updated? (In which case, does this mean we ship different versions of Puppet to different versions of Fedora and call them all PC1?)